packages/block-library/src/site-title/index.js

```diff
@@ -2,17 +2,33 @@
  * WordPress dependencies
  */
 import { mapMarker as icon } from '@wordpress/icons';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
 import metadata from './block.json';
 import edit from './edit';
+import AccessibleSteps from './util/accessible-steps';
 
 const { name } = metadata;
 export { metadata, name };
 
 export const settings = {
+	description: createInterpolateElement(
+		__(
+			'Displays and allows editing the name of the site. The site title usually appears in the browser title bar, in search results, and more. Also available in <path/>.'
+		),
+		{
+			path: (
+				<AccessibleSteps
+					elements={ [ __( 'Settings' ), __( 'General' ) ] }
+					link="options-general.php"
```

Comment:
We'll want to avoid a hardcoded link here since some clients need to be able to override this link. Other places have used a slot/fill pattern, but we may need a new generic filter for links added in block settings.

Added a screencap of feedback to save folks a click, from https://github.com/WordPress/gutenberg/pull/30926

<img width="866" alt="Screen Shot 2021-05-10 at 9 11 15 AM" src="https://user-images.githubusercontent.com/1270189/117690502-c5072c80-b16f-11eb-8aaf-716d254c69f3.png">

Reply:
Thanks for sharing! So we need a way for clients to be able to override internal links. Maybe something for a `[Type] Discussion` issue? 

How do you suggest we go from here?